modules/thunder_media/config/install/responsive_image.styles.media_image.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - image.style.media_image
+    - image.style.media_image_mobile
+    - image.style.media_image_tablet
+#  theme:
+#    - bartik
```

Comment:
I think the responsive image style should not depend on a theme. Couldn't we use "Breakpoint group" "Responsive image" for that?